easybuild/easyconfigs/r/R/R-4.3.1-gfbf-2023a.eb

```
@@ -0,0 +1,374 @@
+name = 'R'
+version = '4.3.1'
+
+homepage = 'https://www.r-project.org/'
+description = """R is a free software environment for statistical computing
+ and graphics."""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['8dd0bf24f1023c6f618c3b317383d291b4a494f40d73b983ac22ffea99e4ba99']
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+    #('Xvfb', '21.1.6'),
+    ('Autotools', '20220317'),
+]
+dependencies = [
+    ('X11', '20230603'),
+    ('Mesa', '23.1.4'),
+    ('libGLU', '9.0.3'),
+    ('cairo', '1.17.8'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.4'),
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.4.2'),
+    ('zlib', '1.2.13'),
+    ('SQLite', '3.42.0'),
+    ('PCRE2', '10.42'),
+    ('libpng', '1.6.39'),  # for plotting in R
+    ('libjpeg-turbo', '2.1.5.1'),  # for plottting in R
+    ('LibTIFF', '4.5.0'),
+    ('Java', '11', '', SYSTEM),
+    ('libgit2', '1.7.0'),
```

A newer `libgit2` was already merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/18606
```suggestion
    ('libgit2', '1.7.1'),
```